File: src/codes/Users_Manual/gain/plane_1.py
~from pyautd3 import EmitIntensity, Plane, Phase
~nx = 1.0
~ny = 0.0
~nz = 0.0
g = Plane([nx, ny, nz])\
        .with_intensity(EmitIntensity.maximum())\
        .with_phase_offset(Phase(0x00))

File: src/codes/Users_Manual/gain/plane_1.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
# let x = 0.;
# let y = 0.;
# let z = 0.;
# let nx = 0.;
# let ny = 0.;
# let nz = 0.;
# let theta = 0.;
let g = Plane::new(Vector3::new(nx, ny, nz))
            .with_intensity(0xFF)
            .with_phase_offset(0x00);
# }